Jenkinsfile

```
@@ -12,7 +12,14 @@ node(){
   }
 
   stage('Deploy')   {
-      cloudFoundryDeploy script:this, deployTool: 'mtaDeployPlugin'
+      def id = new Date().getTime()
+      consumerTestCfSpaceName = "val-it-${id}"
+
+      cloudFoundryCreateSpace script: this, cfSpace: consumerTestCfSpaceName
+      cloudFoundryDeploy script:this,
+                         deployTool: 'mtaDeployPlugin',
+                         cloudFoundry:[ space: consumerTestCfSpaceName]
```

`cloudFoundry/space` is an alias for `space`, [as can be seen in the metadata](https://github.com/SAP/jenkins-library/blob/master/resources/metadata/cloudFoundryDeploy.yaml#L334). Another alias is `cfSpace`:
```suggestion
                         cfSpace: consumerTestCfSpaceName
```